scripts/import/import_all_sv_data.pl

```diff
@@ -343,19 +349,20 @@ sub study_table{
   my $author       = $data->{author};
   my $author_info  = $data->{author};
   my $study_type   = ($data->{study_type}) ? $data->{study_type} : 'NULL';
-  
+
   # Author
   $author_info =~ s/_/ /g;
   my $year_desc = $year->[0];
   if ($author_info =~ /(.+)\set\sal\s(.+)/) {
     $first_author = $1 if (!$first_author);
     $year_desc = $2 if (defined($2));
   }
-        
+
   my $author_desc;
   $author_desc = "$first_author $year_desc " if ($first_author and defined($year_desc));
-  $author = (split('_',$author))[0] if ($author !~ /.+_et_al_.+/);
-  
+  # Don't think this is needed
+  # $author = (split('_',$author))[0] if ($author !~ /.+_et_al_.+/);
```

Comment:
Which values do we expect for $author?

Reply:
It shouldn't be commented on. This line is needed to get the author's name for the description.